public/app/features/explore/Logs.tsx

```diff
@@ -299,12 +314,27 @@ export class UnthemedLogs extends PureComponent<Props, State> {
             <InlineField label="Time" transparent>
               <InlineSwitch value={showTime} onChange={this.onChangeTime} transparent />
             </InlineField>
+            <span
```

Comment:
This shouldn't be hard coded such as this. 

Regarding UI, you can maybe check in #UX slack channel on their thoughts how could we make it clear which toggle belongs to which label. My thought would be to make spacing between label and toggle smaller. And increase the space between the non-related toggle and label.

Reply:
Thanks for the tip, i'll see if i can ask them!